src/main/kotlin/no/nav/syfo/behandler/kafka/dialogmeldingtobehandlerbestilling/DialogmeldingToBehandlerBestillingDTO.kt

```diff
@@ -17,9 +17,9 @@ data class BehandlerDialogmeldingBestillingDTO(
     val dialogmeldingVedlegg: ByteArray? = null,
 )
 
-fun BehandlerDialogmeldingBestillingDTO.toBehandlerDialogmeldingBestilling(
+fun DialogmeldingToBehandlerBestillingDTO.toBehandlerDialogmeldingBestilling(
```

Comment:
Burde mapperen endre navn også?

Reply:
Definitivt. Det blir bedre (men ikke dritbra da, dessverre).

Reply:
Vi trenger flere ord for "til" i språk.

Reply:
https://www.synonymordboka.no/no/?q=til `ad` 🤔

Reply:
DialogmeldingHenimotBehandlerBestilling ruller fint på tunga det 🤔